README.md

```diff
@@ -37,9 +23,26 @@ eks-node-1.20
 eks-gpu-node-1.20
 ```
 
+preinstall.sh installs the prerequisites to install the EKS cluster using terraform. The following packages will get installed:
+- python3
+- awscli
+- aws-iam-authenticator
+- docker (required for DKube install)
+- kubectl
+- terraform
+
+Run the following commands to install the prerequisites.
+```
+# bash preinstall.sh
+# source ~/.bashrc
+```
+
+# Install EKS Cluster
+
 Run the following commands to deploy the EKS cluster and EFS endpoint
 ```
 # bash eks.sh 
+# export KUBECONFIG=$HOME/.dkube/kubeconfig
```

Comment:
It seems to overwrite ~/.kube/config after eks is installed. There should be a way to append and use kubectl set-context